public/app/features/plugins/admin/components/InstallControls/InstallControlsButton.tsx

```diff
@@ -152,7 +152,7 @@ export function InstallControlsButton({
 
     return (
       <HorizontalGroup align="flex-start" width="auto" height="auto">
-        <Button disabled={disableUpdate} onClick={onUpdate}>
+        <Button disabled={disableUpdate} onClick={onUpdate} hidden={plugin.isManaged}>
```

Comment:
why hiding it? (vs not rendering it)

Reply:
No particular reason! I guess it's more common here to not render?

Reply:
yep, just a nit though.